RELEASING.md

```diff
@@ -93,128 +70,101 @@ green, you may create a release as follows:
-   pull repeatedly to confirm that it has received the new docs.
+1. If you feel the error was transient:
+    1. If the error occurred on "Kokoro - Release", selected in step 2, go to [the PR containing your release](https://github.com/googleapis/google-cloud-ruby/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+Release). Unselect the labels "autorelease: published" and "autorelease: failed". This will cause Kokoro to attempt to rebuild and publish the gem you are releasing.
 
-   If the gh-pages branch has not been updated, inspect the Kokoro build logs to
-   confirm that the rake `post` task succeeded.
+    1. If the error occurred on "Kokoro - Linux", "Kokoro - OSx", or "Kokoro - Windows" and you have access to [fusion](https://fusion.corp.google.com/dashboard/findbuilds?search_pattern=google-cloud-ruby%2Fcontinuous&project_types=&include_inactive_projects=false), find the failing build, click on it, then click "rebuild". If you do not have access to [fusion](https://fusion.corp.google.com/dashboard/findbuilds?search_pattern=google-cloud-ruby%2Fcontinuous&project_types=&include_inactive_projects=false), file an [issue](https://github.com/googleapis/google-cloud-ruby/issues/new?template=bug_report.md) containing a link to the build logs, a brief summary, and "@googleapis/yoshi-ruby".
 
-1. Confirm that the gem for the new version is available on
-   [RubyGems.org](https://rubygems.org/gems/google-cloud).
+1. If you feel the error was deterministic:
+    1. If you are confident that the error is unrelated to your release, please open an [issue](https://github.com/googleapis/google-cloud-ruby/issues/new?template=bug_report.md) with a description of the problem and a link to the build logs.
 
-   If RubyGems.org has not been updated, inspect the Circle CI build logs to
-   confirm that the rake `release` task succeeded.
+    1. If the error is related to your release and you have access to the googleapis [rubygems](https://rubygems.org/) account, follow [these instructions](http://help.rubygems.org/kb/gemcutter/removing-a-published-rubygem) to yank the gem. Open an [issue](https://github.com/googleapis/google-cloud-ruby/issues/new?template=bug_report.md) detailing what went wrong and begin working on a fix.
 
-1. Repeat steps 23 through 27 for each gem in your squashed commit.
+    1. If the error is related to your release and you do not have access to the googleapis [rubygems](https://rubygems.org/) account, please open an [issue](https://github.com/googleapis/google-cloud-ruby/issues/new?template=bug_report.md) containing a link to the build logs, a brief summary, and "@googleapis/yoshi-ruby".
 
-1. After the Circle CI master branch build has successfully completed, confirm
-   that the [Kokoro master branch
-   builds](https://github.com/googleapis/google-cloud-ruby/commits/master) are
-   also green by looking for green checkmarks next to the timestamp for each
-   commit.
+## Writing a release manually
```

Comment:
Does the GitHub release actually publish the gem to Rubygems? If so, this is OK, but otherwise maybe change "a release" -> "GitHub Release post" (or similar)